source/declarations.tex

```diff
@@ -2935,8 +2935,12 @@
   using C::x;
   int x(double);
 };
-int f(D* d) {
-  return d->x();    // ambiguous: \tcode{B::x} or \tcode{C::x}
+void f(D* d) {
+   d->x();       // ambiguous: \tcode{x}'s direct parent \tcode{A} is an ambiguous base of the naming class \tcode{D}
```

Comment:
Hmm - still trying to figure these comment threads out - so apologies if this is double spam but here is a snapshot to give you a visual:
![2016-06-09 20_43_31-clipboard](https://cloud.githubusercontent.com/assets/815934/15952040/351e41ae-2e83-11e6-99b3-dc53613c9738.png)